src/app/content/components/Footer.tsx

```diff
@@ -0,0 +1,409 @@
+    @media (min-width: ${desktopMinWidth}em) {
+        font-size: 1.8rem;
+        font-weight: normal;
+        letter-spacing: normal;
+        line-height: normal;
+        line-height: 3rem;
+    }
+`;
+
+// tslint:disable-next-line:variable-name
+const DonateLink = styled.a`
+    ${columnLink}
+    font-weight: bold;
+    text-underline-position: under;
+`;
+
+// tslint:disable-next-line:variable-name
+const FooterLink = styled.a`
+    ${columnLink}
+    text-decoration: none;
+
+    :hover {
+        text-decoration: underline;
+    }
+`;
+
+// tslint:disable-next-line:variable-name
+const BottomLink = styled.a`
+    ${columnLink}
+`;
+
+const column = css`
+    display: grid;
+    grid-gap: 0.5rem;
+`;
+
+// tslint:disable-next-line:variable-name
+const Column1 = styled.div`
+    ${column}
+    grid-area: col1;
+`;
+
+// tslint:disable-next-line:variable-name
+const Column2 = styled.div`
+    ${column}
+    grid-area: col2;
+`;
+
+// tslint:disable-next-line:variable-name
+const Column3 = styled.div`
+    ${column}
+    grid-area: col3;
+`;
+
+// tslint:disable-next-line:variable-name
+const ColumnHeading = styled.div`
+    font-size: 1.8rem;
+    font-weight: bold;
+    letter-spacing: -0.072rem;
+    line-height: normal;
+`;
+
+// tslint:disable-next-line:variable-name
+const FooterBottom = styled.div`
+    font-size: 1.2rem;
+    font-weight: normal;
+    letter-spacing: normal;
+    line-height: normal;
+    background-color: #3b3b3b;
+
+    @media (min-width: ${desktopMinWidth}em) {
+        padding: 2.5rem 0;
+    }
+
+    @media (max-width: ${mobileMinWidth}em) {
+        padding: 1.5rem;
+    }
+`;
+
+// tslint:disable-next-line:variable-name
+const BottomBoxed = styled.div`
+    ${boxed}
+    display: grid;
+    grid-column-gap: 4rem;
+    grid-row-gap: 1.5rem;
+
+    @media (min-width: ${desktopMinWidth}em) {
+        grid-auto-flow: column;
+    }
+
+    @media (max-width: ${mobileMinWidth}em) {
+        padding: 0;
+    }
+`;
+
+// tslint:disable-next-line:variable-name
+const Copyrights = styled.div`
+    display: grid;
+    grid-gap: 1rem;
+`;
+
+// tslint:disable-next-line:variable-name
+const CopyrightDiv = styled.div`
+`;
+
+// tslint:disable-next-line:variable-name
+const Social = styled.div`
+    align-items: center;
+    display: grid;
+    grid-auto-flow: column;
+    grid-gap: 1rem;
+    justify-content: left;
+`;
+
+// tslint:disable-next-line:variable-name
+const SocialIcon = styled.a`
+    ${columnLink}
+    ${textRegularSize}
+    background-color: #959595;
+    align-content: center;
+    border-radius: 50%;
+    display: grid;
+    height: 3rem;
+    justify-content: center;
+    overflow: hidden;
+    width: 3rem;
+`;
+
+// tslint:disable-next-line:variable-name
+const FooterLogo = styled.img`
+    height: 4rem;
+    -webkit-transform: translateY(0.2rem);
+    transform: translateY(0.2rem);
+`;
+
+// tslint:disable-next-line:variable-name
+export class Footer extends Component<any> {
+    public render() {
```

Comment:
can break this render function up into a few chunks so that cc stop complaining about 135 line functions